docs/integrations/argocd/apiops.mdx

```diff
@@ -0,0 +1,329 @@
+
+   ```bash
+   kubectl get pods --namespace argocd
+
+   NAME                                               READY   STATUS    RESTARTS   AGE
+   argocd-application-controller-0                    1/1     Running   0          44s
+   argocd-applicationset-controller-65bb5ff89-lcmbk   1/1     Running   0          45s
+   argocd-dex-server-6f898cbd9-slg8h                  1/1     Running   0          45s
+   argocd-notifications-controller-64bc7c9f7-dgnfm    1/1     Running   0          45s
+   argocd-redis-5df55f45b7-2sf62                      1/1     Running   0          45s
+   argocd-repo-server-74d5f58dc5-dgbnr                0/1     Running   0          45s
+   argocd-server-5b86767ddb-57xlj                     0/1     Running   0          44s
+   ```
+
+1. Log into the Argo CD web UI, first by creating a new port-forwarding session.
+
+   ```bash
+   kubectl port-forward svc/argocd-server -n argocd 8080:443
+   ```
+
+   When you navigate to `http://[YOUR-CLUSTER]:8080`, you'll first see a warning about self-signed certificates, which you can accept to proceed. Finally, Argo CD prompts you to login with a username and password. The username is `admin`, and you can retrieve the automatically-generated administrator password with the following:
+
+   ```bash
+   argocd admin initial-password -n argocd
+   ```
+
+   Once logged in, you'll have access to the Argo CD UI.
+
+   ![Screenshot of Argo CD deployed on Kubernetes](img/argo-setup.png)
+
+1. Finally, log in to Argo CD via the CLI to enable administration.
+
+   ```bash
+   argocd login [YOUR-CLUSTER]:8080
+   ```
+
+## **Step 3**: Set up the demo API {#set-up-demo-api}
+
+Next, you need to set up the Git repository for the API you'll deploy behind your ngrok API gateway. GitOps (and thus APIOps) requires declarative and version-controlled configuration, and that includes the hostname for your deployment. You can't simply clone the demo API repository and apply it to your cluster, as the ngrok-supplied hostname will already be in use.
+
+:::note
+
+If you're curious about the fundamentals of using the ngrok Kubernetes Operator with the Kubernetes Gateway API, such as how to set up your `GatewayClass`, `Gateway`, and `HTTPRoute`, check out our [getting started guide](/docs/k8s/getting-started-gwapi.mdx).
+
+:::
+
+If you have an existing API and GitOps configuration, you're more than welcome to [skip to step 4](#deploy-demo-api) while adopting the Argo CD CLI commands to your Git repository.
+
+1. Create a new ngrok static domain. Navigate to the [**Domains** section](https://dashboard.ngrok.com/cloud-edge/domains) of the ngrok dashboard and click **Create Domain** or **New Domain**. This static domain, which will look like `example.ngrok.app`, will be your `NGROK_DOMAIN` for the remainder of this guide.
+
+1. Fork the repository for the demo API at [joelhans/ngrok-apiops-demo](https://github.com/joelhans/ngrok-apiops-demo).
```

Comment:
we should get this into `ngrok-samples/ngrok-apiops-demo`

Reply:
@stmcallister Done! Thanks for the invite!